libdnf.spec

```diff
@@ -233,21 +233,21 @@ popd
 %check
 %if %{with python2}
 pushd build-py2
-  make ARGS="-V" test
+  make ARGS="-V" test || :
```

Comment:
This (and two following occurrences) looks like reminder of the debugging.

Reply:
Sorry, I forget to remove it